include/zenoh_commons.h

```diff
@@ -683,6 +688,7 @@ struct z_config_t z_config_loan(const struct z_owned_config_t *s);
  * To check if `val` is still valid, you may use `z_X_check(&val)` or `z_check(val)` if your compiler supports `_Generic`, which will return `true` if `val` is valid.
  */
 struct z_owned_config_t z_config_new(void);
+struct z_owned_config_t z_config_null(void);
```

Comment:
What's the purpose of `z_config_null`?

Reply:
Should be made private indeed, but the `null`s are in general there to allow stealing a value. Which reminds me that scouting_config_check is missing